pagination/cursor/api.graphql

```diff
@@ -0,0 +1,27 @@
+type Customer {
+  id:ID!
+  name:String
+  email:String
+}
+
+# Connection types for Customer, which has at least one sortable field
+type CustomerConnection {
+  edges: [CustomerEdge]
+  pageInfo: PageInfo!
+}
+
+type CustomerEdge {
+  node: Customer
+  cursor: String
+}
+
+# Example
+type Query {
+  paginatedCustomers(first: Int != 10 after: String != ""): CustomerConnection
```

Comment:
Make sure prettier is run on the file, then I think we have the somewhat clearer `Int! = 10`